lib/utils.pm

```diff
@@ -536,7 +536,7 @@ sub assert_screen_with_soft_timeout {
     # as in assert_screen
     $args{timeout}             //= 30;
     $args{soft_timeout}        //= 0;
-    $args{soft_failure_reason} //= "$args{bugref}: needle(s) $mustmatch not found within $args{soft_timeout}";
+    $args{soft_failure_reason} //= $args{bugref} . ': needle(s) not found within ' . $args{soft_timeout};
```

Comment:
Why would we not want to have the output of $mustmatch? I can see that it does not work as expected for arrays (yet) but that does not mean it would work for single needle tags. Could you please fix that properly and not just delete?

Reply:
right, it was merge while I was thinking about it but I didn't think so important that info in the record_soft_failure as the needles are visible in the webui and look to me like something to fix soonish before other teams start to see a failure that is not pointing to anything,  but ok, the call is fixed (I could not catch this mistake because is a sporadic error) and we can fix the function itself here: #6891